gateway/webhook/integration_test.go

```diff
@@ -67,8 +79,17 @@ func TestIntegrationWebhook(t *testing.T) {
 		}
 		return nil
 	})
+
 	g.Go(func() (err error) {
-		transformerContainer, err = transformertest.Setup(pool, t)
+		if webhookVersion == "v2" {
+			// To run this test locally with custom repository & image tage, refer the below example.
+			// Example:
+			// 			transformertest.WithRepository("hub.dev-rudder.rudderlabs.com/dockerhub-proxy/rudderstack/develop-rudder-transformer"),
+			// 			transformertest.WithDockerImageTag("feat.sources-v2-spec-controller"),
```

Comment:
Do we need this comment?

Reply:
This helps while running the test cases in local, especially with an ongoing branch.
We can get rid of it as well.

Reply:
Yeah, let's get rid of these comments which are self explanatory.